sftp/1.4/modules/ROOT/pages/sftp-documentation.adoc

```diff
@@ -454,7 +454,7 @@ You can also use a matcher for additional filtering of files.
 | Directory a| String |  The directory in which polled files are contained. |  |
 | Recursive a| Boolean |  Whether or not to also poll files contained in sub directories. |  true |
 | Matcher a| <<matcher>> |  A matcher used to filter events on files that do not meet the file matching criteria. |  |
-| Watermark Enabled a| Boolean |  Controls whether or not to do watermarking, and if so, if the watermark should consider the file's modification or creation timestamps. |  false |
+| Watermark Enabled a| Boolean |  Controls whether or not to do watermarking, and if so, if the watermark should consider the file's modification or creation timestamps. Watermark enabled picks only files that have been created or updated after the last poll was executed. |  false |
```

Comment:
Suggestion: Enable this field to have the connector pick only files that were created or updated after the completion of the latest poll.